src/Containers/CRMTable/CRMTable.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { 
+    CreatedDate,
+    CRMTable, 
+    CustomerProfile,
+    ICRMTableProps, 
+    TableHeaderCell,
+    TagGroup
+} from '../../index';
+import { createStoryTitle } from '../../Constants';
+
+interface FilterProps {
+    column: any;
```

Comment:
not updated, let me know when y ou want me to re-revie wbtw

Reply:
resolved using a built in react-table interface.